websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,160 @@
+const presence = new Presence({
+		clientId: "955350055034945576"
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		browsing: "general.browsing"
+	}),
```

Comment:
why use 3 strings if the rest is in a hardcoded language? You can just remove this IMO